README.md

```diff
@@ -324,10 +324,17 @@ Adds a `<Plane />` that always faces the camera.
 
 #### Environment
```

Comment:
Could you also add the link to the story?

Reply:
The link is there already, it's on line 327

Reply:
How did I miss it? 👁️